src/pages/Proposal/VoteSummary/index.tsx

```diff
@@ -0,0 +1,102 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import { BigNumber } from 'bignumber.js';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { formatCoinsToReadableValue } from 'utilities/common';
+import { generateBscScanUrl } from 'utilities';
+import { useTranslation } from 'translation';
+import { XVS_TOKEN_ID } from 'constants/xvs';
+import { Button, Icon, LabeledInlineContent, EllipseText, Tooltip } from 'components';
+import { ActiveVotingProgress } from 'components/v2/GovernanceProposal/ActiveVotingProgress';
+
+import { useStyles } from './styles';
+
+type VoteFrom = {
+  address: string;
+  voteWeightWei: BigNumber;
+  comment?: string;
+};
+
+interface IVoteSummaryProps {
+  className?: string;
+  votesFrom?: VoteFrom[];
+  // should be passed one of votedForWei/votedAgainstWei/abstainedWei with the votedTotalWei
+  votedForWei?: BigNumber;
+  votedAgainstWei?: BigNumber;
+  abstainedWei?: BigNumber;
+  votedTotalWei?: BigNumber;
+  onClick: () => void;
+  isDisabled?: boolean;
+}
+
+export const VoteSummary = ({
+  className,
+  votesFrom = [],
+  votedForWei,
+  votedAgainstWei,
+  abstainedWei,
+  votedTotalWei,
+  onClick,
+  isDisabled,
+}: IVoteSummaryProps) => {
+  const styles = useStyles();
+  const { t } = useTranslation();
+
+  const getVoteWeight = (voteWeightWei: BigNumber) =>
+    useMemo(
+      () =>
+        formatCoinsToReadableValue({
+          value: voteWeightWei,
+          tokenId: XVS_TOKEN_ID,
+          shortenLargeValue: true,
+          addSymbol: false,
+        }),
+      [],
+    );
+
+  return (
+    <Paper css={styles.root} className={className}>
+      <ActiveVotingProgress
+        votedForWei={votedForWei}
+        votedAgainstWei={votedAgainstWei}
+        abstainedWei={abstainedWei}
+        votedTotalWei={votedTotalWei}
+      />
+      <Button css={styles.button} onClick={onClick} disabled={isDisabled}>
+        {votedForWei && t('vote.for')}
+        {votedAgainstWei && t('vote.against')}
+        {abstainedWei && t('vote.abstain')}
```

Comment:
We can pass this in as a prop called `title` so we don't need to render based on all three votes types. After we clean up the interface we won't have these props